pkg/cluster/bootstrap.go

```diff
@@ -180,61 +314,79 @@ const systemTimeSkew = int64(3)
 // ReconcileBootstrapData is called before any data is saved to the
 // datastore or locally. It checks to see if the contents of the
 // bootstrap data in the datastore is newer than on disk or different
-//  and dependingon where the difference is, the newer data is written
-// to the older.
-func (c *Cluster) ReconcileBootstrapData(ctx context.Context, buf io.ReadSeeker, crb *config.ControlRuntimeBootstrap) error {
+// and depending on where the difference is, the newer data is written
+// to disk or if the disk is newer, the process is stopped and a error
```

Comment:
```
// to disk or if the disk is newer, the process is stopped and a error
```

How does this work with certificate renewal? I thought I remembered that CA cert renewal will cause the certs to get updated on disk, which would then cause subsequent startups to fail because the files on disk are newer than the datastore.

Reply:
Should have no effect on regular rotation but will in fact halt k3s if CAs are rotated on disk since they will be newer than what's in the database.

Reply:
turns out I totally imagined that we currently renew them at all.